Airport Ticket Booking System/Services/ManagerService.BatchUpload.cs

```diff
@@ -0,0 +1,81 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public partial class ManagerService
```

Comment:
We don't usually add dots to the file names :D if this is a big service, just  break it down, and name this `FlightUploadService`. Same for the other services, it's fine to have a service with one method, since each method has a specific responsibility.